test_sets/Defects/6_1/2460.txt

```diff
@@ -0,0 +1 @@
+./test_scripts/Defects/6_1/2460_Resuming_SubscribeWayPoints.lua
```

Comment:
```suggestion
./test_scripts/Defects/7_0/2460_Resuming_SubscribeWayPoints.lua
```
since this wasn't merged in 6.1

Reply:
@jacobkeeler Please notice `develop` branch has been back merged into PR branch and script added to `Defects_release_7_0.txt` test set